include/dao/dao.js

```diff
@@ -322,8 +322,8 @@ module.exports = function DAOModule(pb) {
                 .skip(offset);
 
             //apply sort order
-            if (options.orderBy) {
-                cursor.sort(options.orderBy);
+            if (options.order) {
+                cursor.sort(options.order);
```

Comment:
I worry about switching the name here at such a low level.  This causes a break change to our API.  I am fine with moving to ```order``` instead of ```orderBy``` but we need to support both attributes or risk causing issues to existing deployments.  Maybe introduce a local variable to represent order: ```var order = options.order || options.orderBy;```